scheduler/src/cook/queries.clj

```diff
@@ -28,3 +28,8 @@
             [?e :resource.type/mesos-name ?ident]]
           db)
        (map first)))
+
+(defn get-quota-resource-types
```

Comment:
I don't particularly like this function. Now we have 3 kinds of resources. 

1. Those that go into fenzo without being quota. (disk)
2. Those that go into quota but do not feed fenzo. (rate limits, count)
3. Those that go into quota and fenzo. (mem, cpu gpus)

We have #2 and #3. This code creates #1. 

If you don't give a :resource.type /mesos-name for the resource, the result doesn't show up with queries/get-all-resource-types  so maybe that lets you get rid of this function. 

Alternately, whats wrong with setting a quota for disk as well as everything else? Just have a default quota of infinity and we're good.  @dposada that would get my vote. We allow setting a per user per-pool disk quota. 

Regardless of the above, we should document that distinction carefully, because its getting muddy.

Reply:
Great point. @dposada and I discussed earlier today and came up with the following options for solutions.

1. Leave disk out of quota but add documentation in this function for the different kinds of resources. This is the most simple and we can implement quotas per user at a later time.
2. Use a scalar value for quota (agnostic to type). This is the second simplest and could probably be included in this PR.
3. Make quotas for different types of disks. This is definitely the most complicated, and we are trying to find out from Hyonjee and Sri if this is something that is valuable and worth spending time on. DPO and I agree that it probably wouldn't be included in the scope of this PR.

Let us know your thoughts!

Reply:
I like #1, but we need to write the comment explaining the subtlety of whats going on here and on get-all-resource-types. (explaining the 3 resource types and why they are the way they are.)